live-examples/html-examples/forms/css/meter.css

```diff
@@ -0,0 +1,16 @@
+.meter-example {
+    display: table;
+}
+
+.meter-example div {
+    display: table-row;
+}
+
+label, meter {
```

Comment:
Formatting nit "When using multiple selectors in a rule declarations, give each selector its own line": https://mdn.github.io/mdn-fiori/patterns/css/formatting/#general